.drone.yml

```diff
@@ -1943,48 +1943,58 @@ clone:
 
 steps:
   - name: Set up exec runner storage
+    environment:
+      WORKSPACE_DIR: /tmp/build-darwin-amd64
     commands:
-      - mkdir -p /tmp/build-darwin-amd64
-      - chmod -R u+rw /tmp/build-darwin-amd64
-      - rm -rf /tmp/build-darwin-amd64/go
+      - mkdir -p $WORKSPACE_DIR
+      - chmod -R u+rw $WORKSPACE_DIR
+      - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
 
   - name: Check out code
     environment:
       GITHUB_PRIVATE_KEY:
         from_secret: GITHUB_PRIVATE_KEY
+      WORKSPACE_DIR: /tmp/build-darwin-amd64
     commands:
-      - mkdir -p /tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport
-      - cd /tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport
+      - mkdir -p $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
+      - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
```

Comment:
hmm, wdyt about `cd $WORKSPACE_DIR` and using relative paths everywhere below?

Reply:
I think I've generally tried to avoid that so that it's always clear in what directory something is intended to be run. It makes the file a bit harder for a human to parse (which is part of why I changed from writing the entire path every time, it's really hard to spot errors) but is generally a bit more reliable.